src/forwarders.js

```diff
@@ -572,36 +572,52 @@ export default function Forwarders(mpInstance, kitBlocker) {
     };
 
     this.configureUIEnabledKit = function(configuration) {
-        var newForwarder = null,
+        var newKit = null,
             config = configuration,
-            forwarders = {};
+            kits = {};
 
         // If there are kits inside of mpInstance._Store.SDKConfig.kits, then mParticle is self hosted
         if (!isEmpty(mpInstance._Store.SDKConfig.kits)) {
-            forwarders = mpInstance._Store.SDKConfig.kits;
+            kits = mpInstance._Store.SDKConfig.kits;
             // otherwise mParticle is loaded via script tag
         } else if (!isEmpty(mpInstance._preInit.forwarderConstructors)) {
             mpInstance._preInit.forwarderConstructors.forEach(function(
-                forwarder
+                kitConstructor
             ) {
-                forwarders[forwarder.name] = forwarder;
+                // A suffix is added to a kitConstructor and kit config if there are multiple different
+                // versions of a client kit.  This matches the suffix in the DB.  As an example
+                // the GA4 kit has a client kit and a server side kit which has a client side
+                // component.  They share the same name/module ID in the DB, so we include a
+                // suffix to distinguish them in the kits object.
+                // If a customer wanted simultaneous GA4 client and server connections,
+                // a suffix allows the SDK to distinguish the two.
+                if (kitConstructor.suffix) {
```

Comment:
Does it make sense to check if both name and suffix exist?

Reply:
name will always exist.  it's suffix that is optional.

Reply:
i added suffix as an optional parameter in the IKitConfig.  Will be useful when this file is migrated to TS.